src/random.jl

```diff
@@ -0,0 +1,80 @@
+#=
+Generate random NormalFormGame instances.
+=#
+
+#
+# Random Games Generating
+#
+"""
+    random_game{N}(nums_actions::NTuple{N,Int})
+
+Return a random NormalFormGame instance where the payoffs are drawn
+independently from the uniform distribution on [0, 1).
+
+# Arguements
+* `nums_actions::NTuple{N,Int}`: Tuple of the numbers of actions,
+    one for each player.
+
+# Returns
+* `::NormalFormGame`: The generated N-players random NormalFormGame.
+"""
+function random_game{N}(nums_actions::NTuple{N,Int})
+    if N == 0
+        throw(ArgumentError("nums_actions must be non-empty"))
+    end
+
+    players::NTuple{N,Player{N,Float64}} =
+        ntuple(i -> Player(rand(tuple(nums_actions[i:end]...,
+                                      nums_actions[1:i-1]...))),
+               N)
+
+    return NormalFormGame(players)
+end
+
+#
+# Covariance Games Generating
+#
+"""
+    covariance_game{N, T<:Real}(nums_actions::NTuple{N,Int}, rho::T)
+
+Return a random NormalFormGame instance where the payoff profiles
+are drawn independently from the standard multi-normal with the
+covariance of any pair of payoffs equal to `rho`, as studied in
+[1]_.
```

Comment:
Maybe make it a [footnote](http://docs.julialang.org/en/stable/manual/documentation/#footnotes)?